### PR TITLE
Feature/new button

### PR DESCRIPTION
**Pull request by sabahonarmand**
I've added buttons of pollgram.

### CODE REVIEW

**Review by sadrahkm — APPROVED** [review]

File: src/components/button.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import clsx from "clsx";  
+const useStyle = makeStyles((theme) => ({
+  container: {}
+}));
+
+    function Button(props){
```
Comment (sadrahkm) [inline src/components/button.jsx:8]: In my opinion it's better to use destructuring approach in order to extract property names

Comment (sadrahkm) [inline src/components/button.jsx:8]: In my opinion it's better to use destructuring approach in order to extract property names. Take a look at my codes or parisa's codes

Comment (sadrahkm) [inline src/components/button.jsx:8]: In my opinion it's better to use destructuring approach in order to extract property names. Take a look at my codes or parisa's codes

File: src/components/button.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      return(
+        <div className = "mt-10">
```
Comment (sadrahkm) [inline src/components/button.jsx:10]: In my opinion it's better to use destructuring approach in order to extract property names. Take a look at my codes or parisa's codes

File: src/components/button.jsx (hunk 1 above)
Comment (sadrahkm) [inline src/components/button.jsx:8]: In my opinion it's better to use destructuring approach in order to extract property names. Take a look at my codes or parisa's codes

**Review by parisamhmd — APPROVED** [review]

**Pull request merged**